src/jats/importer/jats-body-dom-parser.ts

```diff
@@ -594,6 +594,135 @@ const nodes: NodeRule[] = [
       }
     },
   },
+  {
+    tag: 'sec[sec-type="affiliations"]',
+    node: 'affiliations_section', // NOTE: higher priority than 'section'
+    getAttrs: (node) => {
+      const element = node as HTMLElement
+
+      return {
+        id: element.getAttribute('id'),
+        category: chooseSectionCategory(element),
+      }
+    },
+  },
+  {
+    tag: 'aff',
+    node: 'affiliation',
+    context: 'affiliations_section/',
+    getAttrs: (node) => {
+      const element = node as HTMLElement
+      const aff = {
+        id: element.getAttribute('id'),
+      } as any
+      const institution = element.getAttribute('institution')
+      if (institution) {
+        aff.institution = institution
+      }
+      const email = element.getAttribute('email')
+      if (email) {
+        aff.bibliographicName = JSON.parse(email)
+      }
+      const department = element.getAttribute('department')
+      if (department) {
+        aff.department = department
+      }
+      const addressLine1 = element.getAttribute('addressLine1')
+      if (addressLine1) {
+        aff.addressLine1 = addressLine1
+      }
+
+      const addressLine2 = element.getAttribute('addressLine2')
+      if (addressLine2) {
+        aff.addressLine2 = addressLine2
+      }
+
+      const addressLine3 = element.getAttribute('addressLine3')
+      if (addressLine3) {
+        aff.addressLine3 = addressLine3
+      }
+
+      const postCode = element.getAttribute('postCode')
+      if (postCode) {
+        aff.postCode = postCode
+      }
+      const country = element.getAttribute('country')
+      if (country) {
+        aff.country = country
+      }
+      return aff
+    },
+  },
+  {
+    tag: 'sec[sec-type="contributors"]',
+    node: 'contributors_section', // NOTE: higher priority than 'section'
+    getAttrs: (node) => {
+      const element = node as HTMLElement
+
+      return {
+        id: element.getAttribute('id'),
+        category: chooseSectionCategory(element),
```

Comment:
Can this be static?

Reply:
Do you mean something like this?
```
getAttrs: () => {
      return {
        id: 'MPSection:contributors',
        category: 'MPSectionCategory:contributors',
      }
    },
    
```

Reply:
If we want to keep the `id` always to be `MPSection:contributors`, I'll have to exclude it from [jats-parser-utils.ts](https://github.com/Atypon-OpenSource/manuscripts-transform/pull/50/files/faaa1649a542d3b990ff8047420b2d76702133da#diff-dbc411c09874e29105b449f718f7e2337ce776db58e43a75fa8133edf67c1630R79), and make it static in [decoder.ts](https://github.com/Atypon-OpenSource/manuscripts-transform/pull/50/files/faaa1649a542d3b990ff8047420b2d76702133da#diff-c89cc32a6cfe692e9c8fb06947554b63975e88b61d2ce89074fb17d891d483a1R866).
And the same thing for the affiliations section. 
That should be easy to do.

Reply:
We don't need the id to be static. I was talking about the category.

Reply:
Done